### DISCUSSION TITLE
Scrum Prep Milestone #1 - First Week

### DISCUSSION THREAD

**Discussion by YanehRuiz** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#50] [Weekly Retrospective]
### Weekly Development Update: First week of Milestone 1

Hello team! 

* Please provide a brief comment detailing what you accomplished this week, including any challenges faced or insights gained. Make sure to submit your updates by the deadline, which is before our upcoming scrum meeting: Friday, September 6, 2024. 

Thank you for your collaboration!

As for my own progress: This week, I've been actively contributing to our project by taking detailed meeting notes, reviewing and approving issues, creating a discussion and rough visual drafts to guide issue creation for our teams, and compiling essential information from developers. You can find more details about these activities in discussions #2, #5, #37, #51, and #58.

**Comment by joel-alvarado** [comment]
I researched and drafted a basic schema for the app which you can view at #54. As for any insights, I'd like to add that this schema is very malleable and new tables can easily be added (such as push notifications, auth sessions, etc). Feedback is appreciated :)

**Comment by Jjulianirizarry24** [comment]
I accomplished the following:

- Completed PR initializing codebase
- Created Mockup for Report Map using Figma
- Set up weekly meetings with Location Management team 
- Made sure everyone on the team had a task.

Due to being so early into the project, I currently have no blockers. I've gained better insights on the vision for the project and I'm ready to keep reviewing technical and design details next week to get the project going on a nice pace.

**Comment by shenied-maldonado** [comment]
I personally just began researching authentication and session management, although I haven't gotten very far. I spent the week making sure the team I'm leading has a topic to focus on and a category of tasks to pick and choose from. We have UI designs coming out soon for peer review, so look out for that!

**Comment by yadrielsamir** [comment]
For this week's tasks, I finished the design for the admin's profile page. I also got reunited with the admin team to set up the weekly meetings and ensure everybody had tasks to complete during the week.

**Comment by RaquelQuinones** [comment]
I created around 5 or 6 tasks to manage the admin login page and address its security aspects. I encountered a problem while trying to install the dependencies on my computer, but aside from that, I've researched login page designs and started working on the design.

**Comment by Alejandro-DeJesus** [comment]
For this week, I organized my team, by running our teams first meeting, and settling matters, such as Goals for milestone 1.  This week I also did research on react, since I'm not familiar with the language.

**Comment by NathBat09** [comment]
For this week, I researched about Flask and PostgreSQL and how to use it for our database. I took part of the meetings to set up our tasks and the objective of this milestone.

**Comment by williammartinez10** [comment]
For this week, I started working with user sign up and log in components, mainly focusing on making them match the proposed wireframe designs.

**Comment by Chosen-Juan1** [comment]
For this week, I set out to sketch a potential design for the admin's communication interface.

**Comment by javier-rivera8** [comment]
This week, I met with my team and divided the tasks to start working on. I also did some research on PostgreSQL, as I haven't worked with it before.

**Comment by Rodal1** [comment]
This week I sketched a potential design for the Admin Interface Settings using Figma, logged all the information of the Admin Interface meeting.

**Comment by johnsantiago9** [comment]
Worked in the creation of the logo and the pallet.  Talked with some partners about how and which color could ve used for it.  Also researched in different platforms where this can be made to obtain a good result.

**Comment by jonVargas1** [comment]
This week I focused on familiarizing myself with Flask and SQL. Additionally, my team and I discussed meeting times and set our team goals to keep ourselves on track.

**Comment by JorgeRivera94** [comment]
This week I read about PostgreSQL in search of what additional tasks could be involved in representing reports in a database, as I have worked with queries before but never implemented a database. Met with the rest of the team to get organized in this first milestone.

**Comment by lex939** [comment]
This week, I organized and uploaded discussions to GitHub, establishing a structured platform for collaboration. I also presented and discussed my role as documentation leader with the group. While there was a delay in posting documentation issues due to a problem with assignees, the issue has now been fixed, and the tasks will be posted later today. Additionally, I assisted in the first Leaders and Managers (LM) meeting, where I spoke about my role and interacted with managers to coordinate our approach for the upcoming weeks of the project.

**Comment by JorgeC08** [comment]
In this week I downloaded the repositories, verified the requirements, and today I was able to join to 3 issues thanks to my managers and team leaders.

**Comment by JuanJRivera** [comment]
I'm working with Noel Padilla on documenting tasks for the User Login Page Design Process and the User Profile Management Page Development. Additionally, I am conducting research on how to implement authentication and session management for the project.

**Comment by evnd04** [comment]
This week, I’ve been collaborating closely with @YanehRuiz, actively contributing to our project by taking thorough meeting notes, reviewing and approving issues, creating discussions and initial visual drafts to guide issue creation for our teams, and gathering key information from developers. I also introduced a difficulty and urgency category for issues, grading them on a scale of 1-5 to better prioritize tasks. My plan for next week is to add the point tracking system.

**Comment by lex939** [comment]
The Scrum prep has started. The Weekly Retrospective for the first week of Milestone 1 is now closed. Please note that any replies after this comment will be considered as late replies.

**Comment by noel-vargas** [comment]
I created a draft design for the profile management page, login and account creation and started creating the components on react-JavaScript for the profile management tab.
